packages/storage/src/core/uploaders/ipfs-uploader.ts

```diff
@@ -357,14 +357,14 @@ export class IpfsUploader implements IStorageUploader<IpfsUploadBatchOptions> {
           "Unauthorized - You don't have permission to use this service.",
         );
       }
-      throw new Error("Failed to upload files to IPFS");
+      throw new Error(`Failed to upload files to IPFS - ${res.status} - ${res.statusText} - ${await res.text()}`);
```

Comment:
There's an uploader for React Native as well, we need to remember to update it as well. 

I'll create a ticket to make a pass on it and will add comments here to remind folks to update the RN one when they update this one.